dockerfiles/mesh-bootstrap/Dockerfile

```diff
@@ -9,7 +9,7 @@ FROM golang:1.12.13-alpine3.9 as mesh-builder
 
 RUN apk update && apk add ca-certificates nodejs-current npm make git dep gcc build-base musl linux-headers
 
-WORKDIR /go/src/github.com/0xProject/0x-mesh
+WORKDIR /0x-mesh
```

Comment:
Why do we use `/` instead of `$HOME`? Is this a docker specific convention?

Reply:
AFAIK It doesn't matter. I don't think there's a specific convention.

Reply:
Also worth noting, in the previous version, _we had_ to make `WORKDIR` inside your `$GOPATH`. So that's why it was originally written that way. Now the only requirement is that it's _outside_ of your `$GOPATH`.